LayoutTests/fast/css/ignore-empty-focus-ring-rects.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
How is this render tree dump test testing the focus ring drawing?

Reply:
I was under same impression that this test shouldn't dump tree text but it did so I used it for expectation. Blink/Chromium uses 'png' for this test case on per platform.

Reply:
Changed to 'reftest' and deleted platform specific expectation files.